Runtime/Shapes/ShapePrimitive.cs

```diff
@@ -1,7 +1,7 @@
 namespace UnityEngine.ProBuilder.Shapes
 {
     [System.Serializable]
-    public abstract class Shape
+    public abstract class ShapePrimitive
```

Comment:
What is the reason for this change? `ShapePrimitive` is redundant - if `Shape` is too vague,  can we use `Primitive` or `Shape3D`

Reply:
I reverted that to Shape :)